### PR TITLE
BullMQ Sample

### PR DESCRIPTION
**Pull request by raphaeltm**
## Samples Checklist
✅

### CODE REVIEW

File: samples/bullmq-bullboard-redis/.github/workflows/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+concurrency:
+  group: deploy
+  cancel-in-progress: false
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
```
Comment (lionello) [inline samples/bullmq-bullboard-redis/.github/workflows/deploy.yaml:17]: needs `contents: read` for private repo support; see #127

Comment (raphaeltm) [reply samples/bullmq-bullboard-redis/.github/workflows/deploy.yaml:17]: Oh. We'll need to change all the samples for that then.

Comment (raphaeltm) [reply samples/bullmq-bullboard-redis/.github/workflows/deploy.yaml:17]: Oh. @lionello I see you did that already.

File: samples/bullmq-bullboard-redis/compose.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  board:
+    build:
+      context: ./board
+      dockerfile: Dockerfile
+    ports:
+      - mode: ingress
+        target: 3000
+        published: 3000
+    environment:
+      - REDIS=redis://redisx.lio.ecs.internal:6379
```
Comment (raphaeltm) [inline samples/bullmq-bullboard-redis/compose.yaml:11]: @lionello these changes look like something that wasn't supposed to be committed. Is that right?

**Comment by raphaeltm** [comment]
Deploys with BYOC now. Merging.

**Pull request merged**